xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js

```diff
@@ -33,6 +33,15 @@ require.config({
 require(['jquery', 'xwiki-meta', 'JobRunner'], function($, xm, JobRunner) {
   'use strict';
   var updateProgress = function(jobUI, job) {
+    if (job.state === 'NONE') {
+      jobUI.find('.ui-progress-background').css('display', 'none');
+      jobUI.find('.ui-progress-message').css('display', 'none');
+    } else {
+      jobUI.find('#state-none-hint').remove();
+      jobUI.find('.ui-progress-background').css('display', 'block');
+      jobUI.find('.ui-progress-message').css('display', 'block');
+    }
```

Comment:
Another option is to use:

    jobUI.find('.ui-progress-background').toggle(job.state !== 'NONE')

See https://api.jquery.com/toggle/#toggle-display .

Reply:
Hm indeed it looks appropriate here.